### ISSUE TITLE
Can't get -t to work

### ISSUE THREAD

**Issue by Sludd** [basildane/WakeOnLAN#202]
Hello,

I can't get the delay/timout to work in scheduler or command line

I've tried setting the computer(s) as WMI and Legacy. Legacy shuts down in 30 seconds and WMI shuts down instantly.

The task schedule looks like this:
-p "C:\ProgramData\Aquila Technology\WakeOnLan\machines.xml" -s -m ComputerName -t 300 -f -c "computer will shutdown in 300 seconds"

And in the command line i've tried
wakeonlanc -s -m ComputerName -t 300

I also experimented with (300), "300", 3000. Placed the -t 300 before and after -s. And tried -s -m computername -t 300 -f -c "test"

It works in the GUI "Timeout (secs)" shutdown method when i have WMI enabled but i need it scheduled.

Any idea what could be wrong?

**Comment by basildane** [comment]
Yes, this is true.  Shutdown is very problematic with windows.  The legacy api does have a time delay parameter, but I think the WMI api does not.
